instructions_scripts/orca_math_create_json_from_docx.py

```diff
@@ -0,0 +1,117 @@
+import re
```

Comment:
It would be nice to include a docstring of the script describing what it is reponsible and move there comments which are below imports.

Reply:
As above - as long as it's working properly, docstring can be updated in future commits.

Reply:
As above - finished with more detailed documentation.